src/data/mock/readonly.ts

```diff
@@ -172,6 +172,8 @@ const generalCharacteristicData: GetAllGeneralCharacteristicsTypes = {
   communityPartnersInvolvedNote: 'frwegqergqgrqwg planContractUpdatedNote',
   geographiesTargeted: true,
   geographiesTargetedTypes: [GeographyType.OTHER],
+  geographiesStatesAndTerritories: [],
+  geographiesRegionTypes: [],
```

Comment:
Not a huge deal, but for these readonly tests, it's useful to always try to populate data for the snapshots